.travis.yml

```diff
@@ -23,16 +25,27 @@ before_script:
 
 cache: pip
 
-deploy:
-    provider: pypi
-    user: aio-libs-bot
-    distributions: sdist bdist_wheel
-    password:
-        # Encrypted with `travis encrypt -r aio-libs/create-aio-app --api-endpoint 'https://api.travis-ci.com/'`:
-        secure: >-
-          ktCdEw9PQO9orA7BmhCn71yOIPK3seomRkJi8+ea54lQsT8cCe+sKZgaXRTBwkcKnKqz11gQl7P11qPNLIWr4cNeVrF6RsDix5YFDxpfqy+x6IPNvpArhkB8/2u2SpFbjzTOh0jT2d4ucSUDV+axjSRPymHK+8lYEMKXucNrzsao8qrExpCMkVJA5rNKeLC537ZN53tVbhwYNih+LsGPXGTvoQoMSQPRk83NfselIper1bPwioUYzIaJCygrOKQKqs/nKOlygMFmPyg4uFoDBh3idG66r8Q9Zmt7DrCAaCxJyGFmjNYHkMzLLh8kl2VuXk+lREbmPDIfNEGzoAM4mX6Jd+LGNvbOOqntRYpW+K4A01qe+YnI142uA7tfxWl1qOqknb0bT6EXKq3iCB4ZXxLdukCcaBAUJP0ctj5wyVpcJ7TAst4SDKLYuZ06I+kx2n51HO5DsfuUQBvkKuMzAHXv0jxXcv8ez2ZS5kBpgQuemNm1NOsrWhtyWzvhlcD2+VMNaYNII63GvAUiwgO8QCf/JNaW12Fdp/nceVUhJWlsmfvQQ1erKqugtecrowh9XWTO5kx7CMrpZJCDdWXMoUMC7aRmJuYaSiz1xCHa44v6xFlMv7bv2UzelRbnpgolCCTzCGXnC6ozct58b/dtGiHKbPJ+cJSOjH513kxRSaM=
-    on:
-        tags: true
-        repo: aio-libs/create-aio-app
-        all_branches: true
-        python: 3.6
+jobs:
+  include:
+    - stage: Upload a new version of python package to PYPI
+      name: Publishing current Git tagged version of dist to PyPI
+      if: repo == "aio-libs/create-aio-app" AND tag IS present
+      script: skip
+
+      before_deploy:
+        - pip install --upgrade pep517
+        - python -m pep517.build --source --binary --out-dir dist .
+        - echo > setup.py
```

Comment:
setup.py is required for deploy stage, so we have to create it before deploy.

Reply:
@Arfey it's a hack. Since we build the dist ourselves, we don't need Travis CI to do this.
But the current dpl implementation calls `python setup.py` unconditionally which effectively overwrites our artifacts.
To prevent this, we're turning this into a no-op.